server/src/main/java/io/druid/indexer/SQLMetadataStorageUpdaterJobHandler.java

```diff
@@ -67,10 +67,10 @@ public Void withHandle(Handle handle) throws Exception
                       .put("created_date", new DateTime().toString())
                       .put("start", segment.getInterval().getStart().toString())
                       .put("end", segment.getInterval().getEnd().toString())
-                      .put("partitioned", (segment.getShardSpec() instanceof NoneShardSpec) ? 0 : 1)
+                      .put("partitioned", (segment.getShardSpec() instanceof NoneShardSpec) ? false : true)
                       .put("version", segment.getVersion())
                       .put("used", true)
-                      .put("payload", mapper.writeValueAsString(segment))
+                      .put("payload", mapper.writeValueAsString(segment).getBytes())
```

Comment:
Can't do that because you didn't specify character encoding.

Can you writeValueAsBytes instead?